cmd/launcher/desktop/desktop.go

```diff
@@ -39,23 +40,12 @@ func handleSignals() {
 
 // continuously monitor for ppid and exit if parent process terminates
 func exitWhenParentGone() {
-	ticker := time.NewTicker(2 * time.Second)
-	defer ticker.Stop()
-
-	f := func() {
-		if os.Getppid() <= 1 {
+	for ; true; <-time.NewTicker(2 * time.Second).C {
+		exists, err := process.PidExists(int32(os.Getppid()))
+		if err != nil || !exists {
 			fmt.Println("parent process is gone, exiting")
```

Comment:
Might not be worth it, but we should probably `logger`,  and not just `fmt.Println`

Reply:
I prefer to kick the can on this one, there is already a lot going on in this PR and I don't see an easy way to pass any logger configs to the new process.